### ISSUE TITLE
[Feature request] Support LTXV2

### ISSUE THREAD

**Issue by kabachuha** [kohya-ss/musubi-tuner#828]
LTXV2 code support has just been merged into ComfyUI!

This is an audio video model, so we will now be able to experiment with the audio lora modalities!

https://github.com/comfyanonymous/ComfyUI/pull/11632

**Comment by kabachuha** [comment]
The checkpoint is not out yet, but the model and sampling code should be enough to begin adding the support

**Comment by kabachuha** [comment]
Out now! https://huggingface.co/Lightricks/LTX-2/tree/main

**Comment by kabachuha** [comment]
Official LTX-2 training code: https://github.com/Lightricks/LTX-2/tree/main/packages/ltx-trainer

**Comment by dorpxam** [comment]
Can we hopefully use this model for small configurations, eg. 16GB VRAM 128GB RAM?

**Comment by kabachuha** [comment]
This model is 19b. Kandinsky 5 is 20b as well and it fits in 5090 (and even 4090) with fp8 and offload, see #774.

**Comment by kohya-ss** [comment]
The LTX-2 is an interesting model, but this repository does not support audio handling, which I think will be quite difficult to implement. 

The official training code seems to be pretty good. It might be a good idea to find out how much VRAM it actually requires and whether it's possible to reduce the amount of VRAM it uses.

**Comment by kabachuha** [comment]
> The official training code seems to be pretty good.

Would be hard to train without musubi block swap at home 🫠

Audio training / loss computation / masked audio is an interesting thing to explore!

**Comment by kabachuha** [comment]
They have the official training code, and the handling of the training loop / loss logic can be derived from there

**Comment by FurkanGozukara** [comment]
I am looking forward to this 100%

**Comment by dorpxam** [comment]
After some experiments, I am pleasantly surprised by the results.

For use in this configuration (RXT-4080 16GB VRAM / 128GB RAM), you can avoid certain problems such as using an FP8 version of the Gemma-3 text encoder and deactivating the ComfyUI live preview to avoid a bug. Even the non-distilled version works quickly, and the results are more than satisfactory for base weights and without more optimization.

The I2V is more delicate to use, it is best to use the compression of the image source to avoid having a frozen image and a simple zoom-in. The T2V work well even if prompt adhesion requires small adjustments.

I'm sorry about the quality of the audio, considering that you do the tests in French language, which is why it has an impact. And so it works well anyway.

I hope the community will work wonders by fine-tuning the model and adding Loras.

I understand that @kohya-ss is not very enthusiastic about the inclusion of this model in this project, but we may benefit from this model in the future, so it will be very cool.

**Comment by kabachuha** [comment]
People are reporting that LoRA can be successfully trained on a single 5090

https://old.reddit.com/r/StableDiffusion/comments/1q6asqd/ltx2_lora_training/

With 8 bit precision, AdamW8bit and 32 lora rank (and maybe with Deepspeed Zero)

Btw, LTX loras are starting to look really cool

https://huggingface.co/oumoumad/LTX-2-19b-LoRA-TEAR/resolve/main/tear_lora_web.mp4?download=true

**Comment by kabachuha** [comment]
Though, when encoding the dataset using the official trainer, I'm stuck on the VAE encode phase because it OOMs

**Comment by maraan666** [comment]
It would be great if we could at least train a character lora on images only in order to achieve character consistency.
I wouldn't be bothered about sound whatsoever.

**Comment by choowkee** [comment]
Would love to have support in Musubi, LTX2 is looking promising. 

Trying my first lora training via AI toolkit for now.

**Comment by BenDes21** [comment]
Hi, any estimation when ltx2 will be available for training musubi ?

**Comment by mrreclusive3-beep** [comment]
just stopping by as i was curious if this was being worked on, yeah, the audio is something new, but if we could bypass that training just so we can do characters that would be nice.  only "working" trainer i can find right now is ai-toolkit, but it needs the full 200gb hf repo, won't use the fp8 checkpoint, or even smaller gemma 3 (i generate with a 400mb unsloth variant), so "working" on a 4090 was it loaded and started, spent about 1 hour for a single epoch of 10 images.
im sure you could make a better trainer, even if we loose audio training, i don't care about training audio anyways.
ai-toolkit has the issue where it wants to load everything together, staging works better for models this large.

**Comment by mrreclusive3-beep** [comment]
for anyone curious, ran some tests today, and as long as you don't need audio training, ltx1 lora's work in ltx2, so im going to train my character lora's on ltx1 until we get ltx2 support.

**Comment by gambikules** [comment]
> It would be great if we could at least train a character lora on images only in order to achieve character consistency. I wouldn't be bothered about sound whatsoever.

Yeah but with LTXv2 you can reproduce her voice. Its a huge +

**Comment by FurkanGozukara** [comment]
@kohya-ss we are looking forward to

this repo working : https://github.com/AkaneTendo25/musubi-tuner/tree/ltx-2

**Comment by FurkanGozukara** [comment]
@kohya-ss we are looking forward to this feature. i just had been notified by another fellow follower

this repo working : https://github.com/AkaneTendo25/musubi-tuner/tree/ltx-2

**Comment by Toserme** [comment]
> [@kohya-ss](https://github.com/kohya-ss) we are looking forward to this feature. i just had been notified by another fellow follower
> 
> this repo working : https://github.com/AkaneTendo25/musubi-tuner/tree/ltx-2

Not sure why you keep tagging Kohya for no reason. Its up to AkaneTendo25 to submit a PR plus it has already been explained weeks ago why it can't be merge as is due to licensing:

https://github.com/AkaneTendo25/musubi-tuner/issues/15#issuecomment-3848116559
